wagtailio/project_styleguide/templates/patterns/_pattern_library_only/comparison_table.html

```diff
@@ -0,0 +1,122 @@
+{# This template provides data for the comparison table, to update it use the streamfield HTML #}
+<tr>
+    <td>
+    </td>
+
+    <td>
```

Comment:
Suggestion: these should be `<th>` if they are part of table header. Can this be added via the table streamfield?

Reply:
Oh dear, I don't think this BE will meet the FE implementation needs

Reply:
@zerolab, what do you think, the "heading" block is part of the TypedTableBlock which contains <th>. In the design, we use an SVG / image.

Reply:
I feel the first row should be `th`s (so that would include the logos)

Reply:
Thanks both.